subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -703,6 +703,60 @@ The TestNG framework has a quite similar concept. In TestNG you can specify diff
             </sample>
 ++++
 
+[[migrate_to_junit5]]
+==== Migrate to JUnit 5
+
+http://junit.org/junit5/docs/current/user-guide[JUnit 5] is the latest version of the well-known `JUnit` test framework.
+Unlike its predecessor, `JUnit 5` is modularized and composed of several modules:
+
+    JUnit 5 = JUnit Platform + JUnit Jupiter + JUnit Vintage
+
+The `JUnit Platform` serves as a foundation for launching testing frameworks on the JVM. `JUnit Jupiter` is the combination of the new http://junit.org/junit5/docs/current/user-guide/#writing-tests[programming model]
+ and http://junit.org/junit5/docs/current/user-guide/#extensions[extension model] for writing tests and extensions in JUnit 5. `JUnit Vintage` provides a `TestEngine` for running JUnit 3 and JUnit 4 based tests on the platform.
+
+The following code enables `JUnit Platform` support in `build.gradle`:
+
+    test {
+        useJUnitPlatform()
+    }
+
+[[junit_jupiter_engine]]
+===== JUnit Jupiter Engine
+
+To enable `JUnit Jupiter` support, add the following dependencies:
+
+++++
+<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="jupiterdependencies" dir="testing/junitplatform/jupiter" title="Jupiter Engine Dependencies">
+    <sourcefile file="build.gradle" snippet="jupiter-dependencies"/>
+</sample>
+++++
+
+Put the following code into `src/test/java`:
+
+++++
+<sample xmlns:xi="http://www.w3.org/2001/XInclude" id="jupiterexample" dir="testing/junitplatform/engine/src/test/java/org/gradle/junitplatform" title="Jupiter Test Example">
```

Comment:
The word "Example" should be lower cased.

Reply:
Good catch, done.